### ISSUE TITLE
As an official I want to be able to embed a map to inquiry (e.g. service map).

### ISSUE THREAD

**Issue by itimo** [City-of-Helsinki/kerrokantasi-ui#42]

**Comment by rambo** [comment]
Is inquiry the same as a hearing ? If so please change the title so we use terms consistently.

**Comment by joonasp** [comment]
the service map here means specifically this Helsinki Service Map: http://www.hel.fi/palvelukarttaws/rest/index_en.html

**Comment by juyrjola** [comment]
No, it means this Service Map: http://palvelukartta.hel.fi/ =)

**Comment by joonasp** [comment]
right. and you had a tool in mind for "drawing" the map polygon as an official? or is that in another ticket?

**Comment by Rikuoja** [comment]
Service map is embeddable, but there is no ready made UI for embedding here.
